mppm/virtual-players/virtual-players-enable.md

```diff
@@ -14,9 +14,9 @@ You must enable **Virtual Players** before you enter **Play mode**.
 To enable **Virtual Players** for **Play mode**, do the following.
 
 1. In the Unity Editor, open the Multiplayer Play Mode window by navigating to **Window** > **Multiplayer Play Mode**.
-2. If you want to use Player Tags for any of your players, select the Player Tag from the **Tag** dropdown associated with each player. Player Tags can only be added to inactive players before entering Play Mode.
+2. If you want to use Player Tags for any of your players, select the Player Tag from the **Tag** dropdown associated with each player. You can only add Player Tags to inactive players before entering Play Mode.
 3. Select the checkbox for each Virtual Player (Player 2, Player 3, or Player 4) you want to add to Play Mode.
 4. Wait for the player status to change from inactive to active for each Virtual Player you selected.
-5. Once active, the Virtual Player(s) appear in Play Mode in the Main Editor window. A separate window opens for each enabled player with the player name as the window title.
+5. When active, the Virtual Player(s) appear in Play Mode in the Main Editor window. A separate window opens for each enabled player with the player name as the window title.
 
 To understand how **Multiplayer Play Mode** works with **Play mode**, refer to [**Synchronize with Play mode**](../synchronize).
```

Comment:
Both this link and the one at the top of the page (create and configure your tags) are broken.